packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -633,6 +634,21 @@ class DragTarget<T extends Object> extends StatefulWidget {
   ///
   /// The builder can build different widgets depending on what is being dragged
   /// into this drag target.
+  ///
+  /// [onWillAccept] or [onWillAcceptWithDetails] is called when a draggable
```

Comment:
```suggestion
  /// The [onWillAccept] or [onWillAcceptWithDetails] is called when a draggable
```

Per https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#use-correct-grammar

Reply:
Does it need to be "The xxx method" then?  I'm not sure how "The" helps, but I also don't have any strong opinions happy to defer to those more active in this section of the code!